src/test/regress/expected/multi_explain.out

```diff
@@ -43,7 +43,7 @@ Distributed Query into pg_merge_job_570000
               ->  Seq Scan on lineitem_290000 lineitem
 Master Query
   ->  Sort
-        Sort Key: (sum(((sum(intermediate_column_570000_1))::bigint)))::bigint, intermediate_column_570000_0
+        Sort Key: COALESCE((sum((COALESCE((sum(intermediate_column_570000_1))::bigint, '0'::bigint))))::bigint, '0'::bigint), intermediate_column_570000_0
```

Comment:
I don't think we really have an alternative, but this will possibly be a bit noticeable performance-wise. Don't think it's bad enough to matter in comparison to the other costs of pulling a lot of rows to the master.